sw/device/tock/dif/build.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+extern crate bindgen;
+
+use bindgen::builder;
+use std::env;
+use std::fs::{self};
+use std::path::Path;
+
+fn main() {
+    let source_root = env::var("MESON_SOURCE_ROOT").unwrap();
```

Comment:
Avoid `unwrap`, since the error it gives is mostly useless, and use `expect` instead.

Reply:
Done.